src/test/scala/cromwell/engine/backend/jes/JesBackendSpec.scala

```diff
@@ -1,13 +1,14 @@
 package cromwell.engine.backend.jes
 
+import java.net.URL
+
 import cromwell.binding.values.{WdlFile, WdlString}
 import cromwell.binding.{Call, CallInputs}
 import cromwell.engine.workflow.WorkflowOptions
-import org.scalatest.mock.MockitoSugar
 import org.scalatest.{FlatSpec, Matchers}
-import spray.json.JsObject
+import org.specs2.mock.Mockito
```

Comment:
Yeah I guess you're right, also pulling specs2 dependency just to mock a `String` and a `URL` may be too much :smile: 
If we were using it as a default everywhere maybe but since we use scalatest I might as well rollback to that.